components/StreamField/StreamField.tsx

```diff
@@ -25,6 +27,16 @@ const StreamField = ({ body, topics }: StreamFieldProps) => {
                         </div>
                     );
                 }
+                if (blockType === 'text') {
+                    return (
+                        <div key={i} className={styles.text}>
+                            <RichTextBlock
+                                key={i}
+                                {...(block as RichTextBlockItem)}
+                            />
+                        </div>
+                    );
+                }
```

Comment:
**Question**

Why can't we add this to the `BLOCKS` object above?

Reply:
The styling for the sections and plain text/paragraphs is different...

Reply:
Right my bad - I didn't notice that the className changed. 

**Suggestion** 

What do you think about adding `<div className={styles.text}>` inside the `RichTextBlock` component. 

If I understand it correctly, with how it's built now, every time you want to render a `RichTextBlock`, you'll also need to declare the `div` that encloses it. (The same applies for all the blocks in `BLOCKS` and `TopicsBlock`)

Reply:
Yes, I preferred styling all those blocks in `Streamfield.module.scss` because they share a lot of styles rather than doing it separately.